### PR TITLE
release notes 2.1.0 and 2.0.1

### PR DESCRIPTION
**Pull request by vicilliar**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
release notes for 2 releases

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# Release 2.1.0
+
+## New Features
```
Comment (pandu-k) [inline RELEASE.md]: `## New Features` ->  `## New features` (style)

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+- Search result maximum limit and offset greatly increased. Maximum `limit` parameter increased from 400 to 1,000, `offset` increased from 1,000 to 10,000. Maximum value for `MARQO_MAX_RETRIEVABLE_DOCS` configuration is now 10,000 (https://github.com/marqo-ai/marqo/pull/735, https://github.com/marqo-ai/marqo/pull/737​​). See search limit and offset usage [here](https://docs.marqo.ai/2.1.0/API-Reference/Search/search/#limit)
```
Comment (pandu-k) [inline RELEASE.md]: ```See search limit and offset usage...``` -> ```See search `limit` and `offset` usage...``` (style)

File: RELEASE.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+
+## Bug fixes and minor changes
+- Improved the Marqo bootstrapping process to address unexpected API behaviour when no index has been created yet (https://github.com/marqo-ai/marqo/pull/730).
+- Addressed a bug where create_index accepted both dependent_fields and dependentFields (https://github.com/marqo-ai/marqo/pull/717, https://github.com/marqo-ai/marqo/pull/734). Using dependent_fields will now raise a 400 validation error.
```
Comment (pandu-k) [inline RELEASE.md]: Is there a way to emphasise the value this fix makes, rather than the negatives

File: RELEASE.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+- Fixed parsing issue for documents in unstructured indexes (https://github.com/marqo-ai/marqo/pull/732).
```
Comment (pandu-k) [inline RELEASE.md]: Again, can we focus on the value this change is bringing to users, rather than the negatives

File: RELEASE.md [hunk 5]
```diff
@@ -1,3 +1,24 @@
+# Release 2.1.0
+
+## New Features
+- Search result maximum limit and offset greatly increased. Maximum `limit` parameter increased from 400 to 1,000, `offset` increased from 1,000 to 10,000. Maximum value for `MARQO_MAX_RETRIEVABLE_DOCS` configuration is now 10,000 (https://github.com/marqo-ai/marqo/pull/735, https://github.com/marqo-ai/marqo/pull/737​​). See search limit and offset usage [here](https://docs.marqo.ai/2.1.0/API-Reference/Search/search/#limit)
+
+## Bug fixes and minor changes
+- Improved the Marqo bootstrapping process to address unexpected API behaviour when no index has been created yet (https://github.com/marqo-ai/marqo/pull/730).
+- Addressed a bug where create_index accepted both dependent_fields and dependentFields (https://github.com/marqo-ai/marqo/pull/717, https://github.com/marqo-ai/marqo/pull/734). Using dependent_fields will now raise a 400 validation error.
+- Fixed parsing issue for documents in unstructured indexes (https://github.com/marqo-ai/marqo/pull/732). 
+- Made vector db config upgrades and rollbacks easier (https://github.com/marqo-ai/marqo/pull/735, https://github.com/marqo-ai/marqo/pull/736).
```
Comment (pandu-k) [inline RELEASE.md]: `db` -> `vector store layer`

File: RELEASE.md [hunk 6]
```diff
@@ -1,3 +1,24 @@
...
+- Made vector db config upgrades and rollbacks easier (https://github.com/marqo-ai/marqo/pull/735, https://github.com/marqo-ai/marqo/pull/736). 
+- Readme improvements (https://github.com/marqo-ai/marqo/pull/729). 
+
+
+# Release 2.0.1
+
+## Bug fixes and minor changes
+- Improved stability of use_existing_tensors feature in `add_documents` (https://github.com/marqo-ai/marqo/pull/725).
+- Improved readability of Marqo start-up logs (https://github.com/marqo-ai/marqo/pull/719).
+- Removed obsolete examples (https://github.com/marqo-ai/marqo/pull/721, https://github.com/marqo-ai/marqo/pull/723/files).
```
Comment (pandu-k) [inline RELEASE.md]: PR 723 doesn't render properly

File: RELEASE.md (hunk 4 above)
Comment (pandu-k) [reply RELEASE.md]: e.g.: "improved validation for ..."

File: RELEASE.md (hunk 3 above)
Comment (pandu-k) [inline RELEASE.md]: Corrections: 
```
Addressed a bug where `create_index` accepted both `dependent_fields` and `dependentFields` as request body parameters (https://github.com/marqo-ai/marqo/pull/717, https://github.com/marqo-ai/marqo/pull/734). Using `dependent_fields` will now raise a 400 validation error.
```

**Review by farshidz — APPROVED** [review]

**Pull request merged**